samples/golf/CMakeLists.txt

```diff
@@ -110,10 +104,6 @@ include_directories(
   ${SOCIAL_INCLUDE_DIR}
   src)
 
-if(USE_NEWSFEED)
-  include_directories(${CURL_INCLUDE_DIR})
```

Comment:
This isn't necessary - the built-in curl module for cmake will sort out the required include directories when you link against the target, as I've done below